pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[tool.black]
+line-length = 119
```

Comment:
Are we sure we want 119? Thoughts on 139 or 159? I personally use 159 in my IDE.

Reply:
120 is the default line length for Intellij. 139/159 will be too long for Python. Actually PEP8 will ask for 79: https://legacy.python.org/dev/peps/pep-0008/#maximum-line-length .  It is reasonable to use 119 for AutoMM.

Reply:
Huggingface uses 119 (https://github.com/huggingface/transformers/blob/main/pyproject.toml#L2)
sklearn uses 88 (https://github.com/scikit-learn/scikit-learn/blob/main/pyproject.toml#L18)
PEP 8 suggests 79. The default line length in Pycharm is 120. Considering all these, I think we'd better not use too long lines.

Reply:
Ok, we can try 119 or 120 and see.